### ISSUE TITLE
Remove todo notes

### ISSUE THREAD

**Issue by Hjdskes** [spoofax-shell/doc#25]
Before we submit the draft, the `\todo` have to be resolved:

```
02-repl.tex:\todo{Make~\cref{table:feature-matrix} complete. At the very least, make the
02-repl.tex:\todo{Is this only Lisp REPLs? Is it \textit{all} Lisp REPLs or just a subset
06-require-ana.tex:\todo{Quite the same as environment inspection maybe?}
```

**Comment by Hjdskes** [comment]
Todonotes in 02-repl.tex have been removed.

**Comment by Hjdskes** [comment]
Fixed in #26.
